src/parser/open-api/v3/navigationParser.js

```diff
@@ -56,5 +58,15 @@ export function getServicesMethod ({ path, method, request, params, responses })
     servicesMethod.parameters = params
   }
 
+  if (servers && servers.length > 0) {
+    servicesMethod.paths = servers.map(server => {
+      return {
+        url: server.url + path,
+        description: server.description
```

Comment:
This will result in there always being a description property, even if it's `undefined`. I'm not sure if that's desired or not?